.github/workflows/lint.yml

```
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 20.17.0
+          node-version: 22
```

Since we've been talking about pinned version granularity: even if we do decide to keep the pin for the test CI, I don't think there is much value in pinning a patch version for the lint CI

---

I agree! In which case, can we point this at the `.node-version` file?

```suggestion
          node-version-file: .node-version
```

---

While I disagree with using the DO buildpack and prefer to know what we're deploying in advance of deploying it so that we can test it, given the DO buildpack situation and the preferences of others I think referencing the .node-version file is nice.